### ISSUE TITLE
missing main.py and config.py

### ISSUE THREAD

**Issue by XiaoshuiHuang** [imkemayer/MissDeepCausal#7]
Thanks for sharing your code. It seems the current version missing "main.py" and "config.py". Would you please upload these two files so that I can test your code.

**Comment by gamazeps** [comment]
Hi !

Since this is a library-ish there is no `main.py` file.
However if you want to reproduce the results you will have to run `exp_template.py`.

The pipeline to reproduce would be:

```
git clone git@github.com:imkemayer/MissDeepCausal.git
cd MissDeepCausal
python3 -m venv venv
pip3 install -r requirements.txt
python exp_template.py
```

By default exp_template will try to compute *all* the [hyper parameter grid](https://github.com/imkemayer/MissDeepCausal/blob/b3ff9de58b65bd45b1137efc6cfeab95cb00d55d/exp_template.py#L68-L91), which will take a few years, you can fix the values with the [different flags](https://github.com/imkemayer/MissDeepCausal/blob/b3ff9de58b65bd45b1137efc6cfeab95cb00d55d/exp_template.py#L22-L41)

**Comment by gamazeps** [comment]
Yeah sorry, the README.md was a bit stale compared to the development, we will update it :)
